pkgs/development/libraries/qtspell/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, enchant
+, libsForQt5
```

```suggestion
, qttools
```
should use `qttools` in `buildInputs` also